pypeit/tests/test_masterframe.py

```diff
@@ -2,28 +2,28 @@
 Module to run tests on armasters
 """
 import os
+
+from IPython import embed
+
 import numpy as np
-import pytest
 
 from astropy.io import fits
 
 from pypeit import masterframe
 from pypeit.images import buildimage
+from pypeit.tests.tstutils import data_path
 
-def data_root():
-    return os.path.join(os.path.dirname(__file__), 'files')
 
 def test_masterframe_methods():
     master_key = 'A_1_01'
```

Comment:
does this need to be `A_1_DET01` ?

Reply:
Changed for consistency, but it doesn't need to be.  `MasterFrame` only sees the key as a string, and doesn't know how it's been defined.